docs/html/installing.rst

```diff
@@ -76,8 +76,8 @@ Install behind a proxy::
 
   python get-pip.py --proxy="http://[user:passwd@]proxy.server:port"
 
-``get-pip.py`` can also be used to install a specified combination of ``pip``,
```

Comment:
This shouldn't probably stay mono space as its naming the package on PyPI, like wheel and setuptools.

Reply:
Should or shouldn't? Are you referring to get-pip.py or pip?